.travis/before-deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+mv ~/.gnupg ~/dot.gnupg  
+
+openssl aes-256-cbc -K $encrypted_24431db6f336_key -iv $encrypted_24431db6f336_iv -in codesign.gpg.enc -out .travis/codesign.gpg -d
```

Comment:
argh, It seems that the `codesign.gpg.enc` parameter is incorrect. missing the `.travis/`

https://travis-ci.org/github/teiid/teiid-language-server/jobs/686143017#L572

`codesign.gpg.enc: No such file or directory`

Reply:
https://github.com/teiid/teiid-language-server/pull/27